src/jit/target.h

```diff
@@ -780,6 +781,7 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
   #define CODE_ALIGN               1       // code alignment requirement
   #define STACK_ALIGN              16      // stack alignment requirement
   #define STACK_ALIGN_SHIFT        3       // Shift-right amount to convert stack size in bytes to size in pointer sized words
+  #define STACK_ALIGN_SHIFT_ALL    4       // Shift-right amount to convert stack size in bytes to size in STACK_ALIGN units
```

Comment:
Yes, it's still used, by legacy JIT. It certainly confused me. I suppose we could delete it from the non-legacy archs in target.h.